### DISCUSSION TITLE
Netlify Serverless Functions

### DISCUSSION THREAD

**Discussion by dungle-scrubs** [ctrl-alt-del-world/midway#33] [Q&A] [answered]
Hi, 

Thanks so much for sharing this template!  I've learned so much from it.

Regarding the readme, it references `[https://YOUR_URL.DOMAIN/.netlify/functions/shopify-sync]` however it looks like you have your functions folder git ignored.

Would you be willing to share that part?  I'm new to serverless functions and hoping to learn how to starting building them for Shopify integrations.

Thanks again

**Comment by Hahlh** [comment, marked as answer]
Hey @dungle-scrubs ! 

They are here:
[https://github.com/ctrl-alt-del-world/midway/tree/master/web/src/lambda](https://github.com/ctrl-alt-del-world/midway/tree/master/web/src/lambda)

**Reply by dungle-scrubs** [reply]
Thanks!
